package/Dockerfile.shipyard-dapper-base

```
@@ -3,6 +3,7 @@ FROM fedora:33
 # Unless specified otherwise, compress to a medium level which gives (from experemintation) a
 # good balance between compression time and resulting image size.
 ARG UPX_LEVEL=-5
+ARG CUTTING_EDGE=devel
```

Won't it make more sense?

```suggestion
ARG SUBCTL_VERSION=devel
```